src/core/sprites/Sprite.js

```diff
@@ -522,7 +522,7 @@ Sprite.prototype.destroy = function (destroyTexture, destroyBaseTexture)
 
     if (destroyTexture)
     {
-        this._texture.destroy(destroyBaseTexture);
+        this._texture.destroy(destroyBaseTexture === undefined ? true : destroyBaseTexture);
```

Comment:
The problem that memory is not free up when call Container.destroy because there no way to set destroyBaseTexture parameter and by default it treat as false.

There are two options: add second parameter to all destroy function or change destroyBaseTexture default value. While I looking through code found Text.destroy and because it simplest change I decided to do it, but personally I prefer 2nd parameter change here.

How docs should be updated? Should Text.destroy be updated too?